ipyfilechooser/filechooser.py

```diff
@@ -12,26 +14,26 @@ class FileChooser(VBox, ValueWidget):
 
     def __init__(
             self,
-            path=os.getcwd(),
-            filename='',
-            title='',
-            select_desc='Select',
-            change_desc='Change',
-            show_hidden=False,
-            select_default=False,
-            use_dir_icons=False,
-            show_only_dirs=False,
-            filter_pattern=None,
+            path: str=os.getcwd(),
+            filename: str='',
+            title: str='',
+            select_desc: str='Select',
+            change_desc: str='Change',
+            show_hidden: bool=False,
+            select_default: bool=False,
+            use_dir_icons: bool=False,
+            show_only_dirs: bool=False,
+            filter_pattern: Optional[Sequence[str]]=None,
             **kwargs):
         """Initialize FileChooser object."""
         self._default_path = path.rstrip(os.path.sep)
         self._default_filename = filename
-        self._selected_path = None
-        self._selected_filename = None
+        self._selected_path: Optional[str] = None
```

Comment:
Can you please provide some context why typing hints are required for internal variables? These variables are not exposed or meant to be used outside this class.